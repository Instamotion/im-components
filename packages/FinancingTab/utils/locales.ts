const de = {
  'default.financing_tab.more_financing_info': 'Mehr zu Finanzierungsarten',
  'default.financing_tab.monthly_rates': 'Monatsraten',
  'default.financing_tab.deposit': 'Anzahlung {minMax}',
  'default.financing_tab.final_installment': 'Schlussrate {minMax}',
  'default.financing_tab.easygo1': 'Niedrige monatliche Raten​',
  'default.financing_tab.easygo2': 'Bis zu 61 Monate Laufzeit​',
  'default.financing_tab.easygo3': 'Schlussrate oder Refinanzierung am Ende der Laufzeit​',
  'default.financing_tab.classic1': 'Ohne Schlussrate​',
  'default.financing_tab.classic2': 'Bis zu 96 Monate Laufzeit​',
  'default.financing_tab.classic3': 'Kredit nach Laufzeit getilgt​',
  'default.financing_tab.details': 'Einzelheiten',
  'default.financing_tab.rates': '{rates} monatliche Raten á',
  'default.financing_tab.monthly_rate': '{rate} € / Monat',
  'default.financing_tab.adjust_financing': 'Finanzierung anpassen',
  'default.financing_tab.reset_financing': 'Finanzierung zurücksetzen'
};

export default { de };
