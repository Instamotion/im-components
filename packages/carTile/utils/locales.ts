const de = {
  'car.tile.not_available': 'Nicht verf√ºgbar',
  'car.tile.or': 'oder',
  'car.tile.mileage': '{val} km',
  'car.tile.power': '{kw} kW/{ps} PS',
  'car.tile.firstRegistration': '{val}',
  'car.tile.consumptionCombined': '{val}  l/100km (komb.)*',
  'car.tile.co2': '{val} g/km (comb.)*'
};

export default { de };
