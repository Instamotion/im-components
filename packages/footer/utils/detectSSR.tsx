const detectSSR = (): { isBrowser: boolean; isServer: boolean } => {
  const canUseDOM = !!(
    typeof window !== 'undefined' &&
    window.document &&
    window.document.createElement
  );

  return {
    isBrowser: canUseDOM,
    isServer: !canUseDOM
  };
};

export default detectSSR;
