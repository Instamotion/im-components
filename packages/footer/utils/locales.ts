/* eslint-disable */
const de = {
  'checkout.footer.about_us': 'Über uns',
  'checkout.footer.agb': 'AGB',
  'checkout.footer.imprint': 'Impressum',
  'checkout.footer.privacy': 'Datenschutz',
  'checkout.footer.faq': 'FAQ',
  'default.footer.newsletter.title': 'Verpasse mit unserem Newsletter \nkein Angebot mehr!',
  'default.footer.newsletter.linkText': '\nDatenschutzbestimmungen.',
  'default.footer.newsletter.subtitle': ' Mit dem Absenden akzeptierst du unsere ',
  'default.footer.copyrights.text': 'Eine Beteiligungsgesellschaft der',
  'default.footer.menu.automobile': 'Auto',
  'default.footer.menu.brands': 'Marken',
  'default.footer.menu.vehicle_types': 'Fahrzeugtypen',
  'default.footer.menu.gearbox': 'Getriebeart',
  'default.footer.menu.fuel_types': 'Kraftstoffarten',
  'default.footer.menu.year': 'Erstzulassung',
  'default.footer.menu.day_registrations': 'Tageszulassungen',
  'default.footer.menu.jahreswagen': 'Jahreswagen',
  'default.footer.menu.to_dye': 'Farben',
  'default.footer.menu.germany': 'Deutschland',
  'default.footer.menu.instamotion': 'Instamotion',
  'default.footer.menu.about_us': 'Über uns',
  'default.footer.menu.press': 'Presse',
  'default.footer.menu.faq': 'FAQ’s',
  'default.footer.menu.how_it_works': 'So funkiontiert’s',
  'default.footer.menu.blog': 'Blog',
  'default.footer.menu.jobs': 'Jobs',
  'default.footer.legal': 'Rechtliches',
  'default.footer.imprint': 'Impressum',
  'default.footer.legal_notice': 'Rechtliche Hinweise',
  'default.footer.conditions': 'AGB',
  'default.footer.data_protection': 'Datenschutz',
  'default.footer.shipping_and_payment': 'Versand und Zahlung',
  'default.footer.cancellation': 'Widerrufsbelehrung',
  'default.footer.services': 'Services',
  'default.footer.warranty': 'Garantie',
  'default.footer.financing_or_leasing': 'Finanzierung oder Leasing?',
  'default.footer.guidebook_family_car': 'Ratgeber Familienauto',
  'default.footer.advisor_novice_driver': 'Ratgeber Fahranfänger',
  'default.footer.part_exchange': 'Inzahlungnahme',
  'default.footer.guide_for_owners': 'Ratgeber für Hundebesitzer',
  'default.footer.fold_us': 'Foldge uns',
  'default.footer.free_advice': 'Kostenlose Beratung',
  'default.footer.guide_funding': 'Ratgeber Finanzierung',
  'default.footer.guide_leasing': 'Ratgeber Leasing',
  'default.footer.advice_leasing': 'Ratgeber Leasing oder',
  'default.footer.financing': 'Finanzierung',
  'checkout.finePrint':
    '* Weitere Informationen zum offiziellen Kraftstoffverbrauch und den offiziellen spezifischen CO2-Emissionen neuer Personenkraftwagen können dem "Leitfaden über den Kraftstoffverbrauch, die CO2-Emissionen und den Stromverbrauch neuer Personenkraftwagen" entnommen werden, der an allen Verkaufsstellen und bei der Deutsche Automobil Treuhand GmbH (DAT), Hellmuth-Hirth-Straße 1, 73760 Ostfildern-Scharnhausen bzw. im Internet unter <a href="www.dat.de/angebote/verlagsprodukte/leitfaden-kraftstoffverbrauch.html" target="_blank">www.dat.de/angebote/verlagsprodukte/leitfaden-kraftstoffverbrauch.html</a> unentgeltlich erhältlich ist.'
};

export default { de };
/* eslint-enable */
