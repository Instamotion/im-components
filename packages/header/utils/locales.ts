const de = {
  'header.menu.autos': 'Autos',
  'header.menu.top_offers': 'Topangebote',
  'header.menu.wish_list': 'Merkliste',
  'header.menu.how_it_works': "So funktioniert's",
  'header.menu.services': 'Deine Vorteile',
  'header.menu.phone_number': '089 411151-100',
  'header.menu.financing': 'Finanzierung',
  'header.menu.delivery': 'Lieferung',
  'header.menu.warranty': 'Garantie',
  'header.menu.quality': 'QualitÃ¤t',
  'header.menu.additional_services': 'Unsere Zusatzleistungen'
};

export default { de };
