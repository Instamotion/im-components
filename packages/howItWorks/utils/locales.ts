const de = {
  'how_it_works.heading': 'Kaufprozess im Detail',
  'how_it_works.step_1': 'Fahrzeugauswahl',
  'how_it_works.step_2': 'Bezahlung',
  'how_it_works.step_3': 'Zulassung',
  'how_it_works.step_4': 'Auslieferung',
  'how_it_works.tooltip_1': 'Du wählst das passende Fahrzeug aus.',
  'how_it_works.tooltip_2': 'Du wählst zwischen Einmalzahlung oder Finanzierung.',
  'how_it_works.tooltip_3': 'Wir lassen das Auto auf dich zu.',
  'how_it_works.tooltip_4': 'Wir liefern das Auto vor deine Haustür.',
  'how_it_works.more': 'Mehr Info'
};

export default { de };
