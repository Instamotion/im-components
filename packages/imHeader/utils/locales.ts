const de = {
  'header.menu.alle.autos': 'Alle Autos',
  'header.menu.favorites': 'Favoriten',
  'header.menu.faq': 'FAQ',
  'header.menu.top_offers': 'Topangebote',
  'header.menu.how_it_works': "So funktioniert's"
};

export default { de };
