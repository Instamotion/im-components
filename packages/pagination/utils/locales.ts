const de = {
  'pagination.previous': 'Zur√ºck',
  'pagination.next': 'Weiter',
};

export default { de };
