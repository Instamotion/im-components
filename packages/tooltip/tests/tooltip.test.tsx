/* eslint-env jest */
import React from 'react';
import { mount } from 'enzyme';
import { renderWithThemeAndI18n } from '@im-ui/utils';
import toJson from 'enzyme-to-json';
import Tooltip from '../src';

describe('Tooltip', () => {
  it('renders', () => {
    const wrapper = mount(renderWithThemeAndI18n(<Tooltip icon={<i />}>Test</Tooltip>));
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
