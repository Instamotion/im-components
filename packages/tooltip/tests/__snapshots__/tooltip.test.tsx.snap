// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders 1`] = `
<ThemeProvider
  theme={
    Object {
      "button": Object {
        "primary": Object {
          "active": Object {
            "shadow": "inset 0px 0.125rem 0.125rem rgba(0, 0, 0, 0.05);",
          },
          "background": "#75dfc0",
          "border": Object {
            "color": "#75dfc0",
            "radius": 4,
            "width": 0,
          },
          "color": "#313330",
          "disabled": Object {
            "opacity": 0.5,
          },
          "font": Object {
            "family": "Roboto, sans-serif",
            "size": 16,
            "spacing": 0,
            "weight": 600,
          },
          "hover": Object {
            "shadow": "0 0.25rem 0.25rem rgba(0, 0 ,0, 0.1)",
          },
        },
        "secondary": Object {
          "active": Object {
            "shadow": "inset 0px 0.125rem 0.125rem rgba(0, 0, 0, 0.05);",
          },
          "background": "#fff",
          "border": Object {
            "color": "#75dfc0",
            "radius": 4,
            "width": 1,
          },
          "color": "#75dfc0",
          "disabled": Object {
            "opacity": 0.5,
          },
          "font": Object {
            "family": "Roboto, sans-serif",
            "size": 16,
            "spacing": 0,
            "weight": 600,
          },
          "hover": Object {
            "shadow": "0 0.25rem 0.25rem rgba(0, 0 ,0, 0.1)",
          },
        },
      },
      "card": Object {
        "border": Object {
          "radius": 4,
        },
        "shadow": "0 0.25rem 0.25rem 0 rgba(0, 0, 0, 0.1)",
      },
      "color": Object {
        "brightGrey": "#606164",
        "downy": "#75DFC0",
        "flamePea": "#B75C50",
        "lightGrey": "#FAFAFA",
        "niagara": "#20AD90",
        "oil": "#313330",
        "parisDaisy": "#FFE44C",
        "silver": "#C4C4C4",
        "white": "#FFFFFF",
      },
      "font": Object {
        "sans": Object {
          "family": "Roboto, sans-serif",
        },
      },
      "footer": Object {
        "maxWidth": 1320,
      },
      "heading": Object {
        "l": Object {
          "size": 36,
        },
        "m": Object {
          "size": 32,
        },
        "s": Object {
          "size": 24,
        },
        "xl": Object {
          "size": 48,
        },
        "xs": Object {
          "size": 20,
        },
        "xxl": Object {
          "size": 60,
        },
        "xxs": Object {
          "size": 18,
        },
      },
      "input": Object {
        "border": Object {
          "color": "#C4C4C4",
          "width": 1,
        },
        "font": Object {
          "family": "Roboto, sans-serif",
          "size": 16,
        },
      },
      "label": Object {
        "font": Object {
          "family": "Roboto, sans-serif",
          "size": 1,
          "weight": 600,
        },
        "letter": Object {
          "spacing": 0.01,
        },
        "line": Object {
          "height": 1.15,
        },
      },
      "mediaQueries": Object {
        "whenDesktop": "@media (min-width: 1024px)",
        "whenDesktopXL": "@media (min-width: 1280px)",
        "whenMobileL": "@media (min-width: 320px)",
        "whenTablet": "@media (min-width: 768px)",
      },
      "name": "default",
      "table": Object {
        "body": Object {
          "background": "#FFFFFF",
          "font": Object {
            "family": "Roboto, sans-serif",
            "height": 28,
            "size": 12,
            "spacing": 0.4,
          },
        },
        "foot": Object {
          "background": "#FFFFFF",
          "font": Object {
            "family": "Roboto, sans-serif",
            "height": 28,
            "size": 12,
            "spacing": 0.4,
            "weight": 500,
          },
        },
        "head": Object {
          "background": "#FFFFFF",
          "font": Object {
            "family": "Roboto, sans-serif",
            "height": 28,
            "size": 12,
            "spacing": 0.4,
            "weight": 500,
          },
        },
        "innerBorder": Object {
          "color": "#C4C4C4",
          "radius": 0,
          "width": 1,
        },
        "outerBorder": Object {
          "color": "#C4C4C4",
          "radius": 4,
          "width": 2,
        },
      },
    }
  }
>
  <IntlProvider
    defaultFormats={Object {}}
    defaultLocale="en"
    formats={Object {}}
    locale="de"
    messages={Object {}}
    onError={[Function]}
    textComponent={Symbol(react.fragment)}
  >
    <Component
      icon={<i />}
    >
      <styled.div>
        <StyledComponent
          forwardedComponent={
            Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "sc-bwzfXH",
                "isStatic": false,
                "lastClassName": "eYTwFP",
                "rules": Array [
                  "
  position: relative;
  display: inline-block;

  :hover ",
                  ".sc-bdVaJa",
                  " {
    display: block;
  }
",
                ],
              },
              "displayName": "styled.div",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "sc-bwzfXH",
              "target": "div",
              "toString": [Function],
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            }
          }
          forwardedRef={null}
        >
          <div
            className="sc-bwzfXH eYTwFP"
          >
            <i />
            <styled.div>
              <StyledComponent
                forwardedComponent={
                  Object {
                    "$$typeof": Symbol(react.forward_ref),
                    "attrs": Array [],
                    "componentStyle": ComponentStyle {
                      "componentId": "sc-bdVaJa",
                      "isStatic": false,
                      "lastClassName": "gLKrLC",
                      "rules": Array [
                        "
  min-width: 12rem;
  max-width: 22rem;
  color: ",
                        "#313330",
                        ";
  border: solid 1px ",
                        "#20AD90",
                        ";
  padding: 1rem;
  position: absolute;
  top: 50%;
  left: 100%;
  margin-left: 12px;
  display: none;
  background-color: ",
                        "#FFFFFF",
                        ";

  &::before {
    content: '';
    position: absolute;
    top: -1px;
    left: -11px;

    width: 0;
    height: 0;
    border-style: solid;
    border-width: 0 10px 10px 0;
    border-color: transparent ",
                        "#20AD90",
                        " transparent transparent;
  }
",
                      ],
                    },
                    "displayName": "styled.div",
                    "foldedComponentIds": Array [],
                    "render": [Function],
                    "styledComponentId": "sc-bdVaJa",
                    "target": "div",
                    "toString": [Function],
                    "warnTooManyClasses": [Function],
                    "withComponent": [Function],
                  }
                }
                forwardedRef={null}
              >
                <div
                  className="sc-bdVaJa gLKrLC"
                >
                  Test
                </div>
              </StyledComponent>
            </styled.div>
          </div>
        </StyledComponent>
      </styled.div>
    </Component>
  </IntlProvider>
</ThemeProvider>
`;
